lib/spoom/sorbet/sigils.rb

```diff
@@ -28,7 +28,7 @@ module Sigils
         T::Array[String],
       )
 
-      SIGIL_REGEXP = T.let(/^#[\ t]*typed[\ t]*:[ \t]*(\w*)[ \t]*/, Regexp)
+      SIGIL_REGEXP = T.let(/^#[\ t]*typed[\ t]*:[ \t]*(\S*)/, Regexp)
```

Comment:
This original regexp seems of, `[\ t]` would not match tabs as expected. It should be changed to `[[:blank:]]`.

Reply:
I agree. Sorbet's comment parsing also does not seem to support any whitespace between `typed` and `:`. What do you think about this?

`/^#[[:blank:]]*typed:[[:blank:]]*(\S*)/`